frontend/components/App.js

```diff
@@ -289,18 +289,26 @@ export default class App extends React.Component {
           </section>
 
           <section>
-            <h1>Lumen distribution</h1>
+            <h1>LUMEN SUPPLY</h1>
             <div className="mui-col-md-4">
-              <DistributionProgress horizonLiveURL={horizonLive} />
```

Comment:
Are these components (`DistributionProgress`, `TotalCoins`, `LumensAvailable`) used anywhere else? Would be great to delete the code if not.

Reply:
The only thing that is not being used is `DistributionProgress`, which is a graph component. I wasn't sure about deleting it because we might use a graph again in the future?

Reply:
we don't plan on using that again

Reply:
Updated @vcarl